openedx/features/course_experience/waffle.py

```diff
@@ -0,0 +1,66 @@
+"""
+This module contains various configuration settings via
+waffle switches for the course experience app.
+"""
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from openedx.core.djangoapps.site_configuration.models import SiteConfiguration
+from openedx.core.djangoapps.theming.helpers import get_current_site
+from openedx.core.djangoapps.waffle_utils import CourseWaffleFlag, WaffleFlagNamespace, WaffleSwitchNamespace
+
+# Namespace
+WAFFLE_NAMESPACE = 'course_experience'
+
+# Switches
+# Full name course_experience.enable_new_course_outline
+# Enables the UI changes to the course outline for all courses
+ENABLE_NEW_COURSE_OUTLINE = 'enable_new_course_outline'
+
+# Full name course_experience.enable_new_course_outline_for_course
+# Enables the UI changes to the course outline for a course
+ENABLE_NEW_COURSE_OUTLINE_FOR_COURSE = 'enable_new_course_outline_for_course'
+
+# Full name course_experience.enable_site_new_course_outline
+# Enables the UI changes to the course outline for a site configuration
+ENABLE_SITE_NEW_COURSE_OUTLINE = 'enable_site_new_course_outline'
+
+def waffle():
+    """
+    Returns the namespaced, cached, audited Waffle class for course experience.
+    """
+    return WaffleSwitchNamespace(name=WAFFLE_NAMESPACE, log_prefix='course_experience: ')
+
+
+def waffle_flag():
+    """
+    Returns the namespaced, cached, audited Waffle flags dictionary for Completion.
+    """
+    namespace = WaffleFlagNamespace(name=WAFFLE_NAMESPACE, log_prefix=u'course_experience: ')
+    return {
```

Comment:
I thought Greg fixed this, but I guess not: this doesn't have to return a dictionary.  Just return the `CourseWaffleFlag` object.